.gitlab-ci.yml

```diff
@@ -708,6 +708,18 @@ variables:
 .on_testing_cleanup:
   - <<: *if_testing_cleanup
 
+.on_security_agent_changes_or_manual:
+  - changes:
```

Comment:
There are some limitations to this: https://docs.gitlab.com/ee/ci/jobs/job_control.html#jobs-or-pipelines-run-unexpectedly-when-using-changes it will always run on a new branch, and the changes will only be taken with respect to the previous commit. This means that a if you do several commits, and the last one does not modify these files, we can easily miss a failure

Reply:
I think this is fair. On the other hand, currently those jobs are only triggered manually (and often not at all). This is also what is used for system-probe tests

Reply:
If they are currently manual then I guess it is okay, yes 👍